docs/source/pages/lightning.rst

```diff
@@ -11,6 +11,11 @@ TorchMetrics in PyTorch Lightning
 
 TorchMetrics was originaly created as part of `PyTorch Lightning <https://github.com/PyTorchLightning/pytorch-lightning>`_, a powerful deep learning research framework designed for scaling models without boilerplate.
 
+..note::
+    TorchMetrics version v0.8 or higher officially only supports v1.5 and higher of Pytorch Lightning, but we recommend to always keeping both
```

Comment:
Should we really hardcode this in out docs? Shouldn't we just set a minimum requirement version? Just afraid this might fall out of sync...

Reply:
Agree that we run the risk of the text being out of sync in the future. However, I think it is important that we somewhere explicitly in the docs state this kind of comparability (at least for the next couple of releases)

Reply:
Can we make it more generic then? Say something like we aim to always offer compatibility with the last 2 major PL versions?

Reply:
will do